packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -450,6 +458,22 @@ export function exportSimpleSchema(storedSchema: TreeStoredSchema): SimpleTreeSc
 	};
 }
 
+/**
+ * A way to schema in the persisted format from {@link extractPersistedSchema}.
+ * @remarks
+ * The output from this is the same as the output from {@link ITreeAlpha.exportSimpleSchema},
+ * but gets the schema from the caller instead of from an existing tree.
```

Comment:
mis-type in the overall first comment block makes me not totally sure how to interpret this. I'd guess it's something like this?

```suggestion
 * This round-trips data returned by {@link extractPersistedSchema} back into `SimpleTreeSchema`.
 * @remarks
 * The output format from this is the same as the output from {@link ITreeAlpha.exportSimpleSchema},
 * but it gets the schema data directly from the caller instead of from an existing tree.
```

...there are technically nuances with the top-level docs there around the persisted format perhaps not encoding all of the data if you pick a "too old FluidClientVersion" when you write it with `extractPersistedSchema`, but I think if we want to call those out it's better to do so on `extractPersistedSchema` and/or places we allow people to specify such versions for more production-oriented code, such as export APIs + tree construction.

Reply:
feel free to not take that suggestion and rewrite how you want to make it more clear. Just calling it out as something to improve.

Reply:
extractPersistedSchema is documented to to export the:

> "persisted" schema subset of the provided `schema`

Which subset that contains may depend on the version, but I think its well specified.

The package API does not expose any of the in memory stored schema formats, so nothing we support is round tripping stored schema: extractPersistedSchema is always lossy.

Reply:
```suggestion
 * A way to parse schema in the persisted format from {@link extractPersistedSchema}.
 * @remarks
 * The output from this is the same as the output from {@link ITreeAlpha.exportSimpleSchema},
 * but it gets the schema from the caller instead of from an existing tree.
```

I specifically did not say "output format" and just said "output" since this logically does exactly the same thing as opening an existing tree and using ITreeAlpha.exportSimpleSchema, except that you can't use ITreeAlpha.exportSimpleSchema if don't have a full tree. If you duc the schema out of a persisted tree, persistedToSimpleSchema could be applied to it and give the same data (in the same format) exportSimpleSchema would give.

Reply:
The point is that loading a document that used a given stored schema and calling ITreeAlpha.exportSimpleSchema will give the same results as loading the same stored schema with this API.

This means if you want to replicate that behavior but don't have a full document you can load, you can use this API.

If we are going to have a data format which can be loaded, it just seems silly to make it impossible to load unless you have a fully document which its part of, especially with extractPersistedSchema documenting its useful for testing but has no guaranteed format. If someone wants to investigate a failing schema snapshot test, and see how that schema change would impact real use via ITree, this API lets them do that.

I'm trying to make it clear that saving a document to a fluid service then loading int into an ITree and calling ITreeAlpha.exportSimpleSchema is analogous to and persistedToSimpleSchema, except that later works with just the schema )and does not require you to come up with tree contents and a fluid service to put it all in).

I have updated the comment. Do you think thats better?

Reply:
Yeah, the follow-up remark is very helpful. I still think the notion of "same" is underspecified (IMO the second paragraph in the remarks block is perfect as a standalone), but it's less important now as it's more context-setting than anything.